metals/src/main/scala/scala/meta/internal/metals/IdentifierIndex.scala

```diff
@@ -41,9 +42,19 @@ class IdentifierIndex {
   ): Iterable[String] = {
     val identifiers = Set.newBuilder[String]
 
+    def foreach(scanner: LegacyScanner)(f: LegacyTokenData => Unit): Unit = {
```

Comment:
Maybe extract it somewhere, if possible.

Reply:
Done! I was a bit lazy before